pkgs/tools/filesystems/grive2/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, pkgconfig, fetchurl, yajl, cmake, libgcrypt, curl, expat, boost, binutils }:
+
+stdenv.mkDerivation rec {
+  version = "0.5.0";
+  name = "grive2-${version}";
+
+  src = fetchFromGitHub {
+    owner = "vitalif";
+    repo = "grive2";
+    rev =  "ae06eccb38b2fe250c9ddeac3e3973f80b8a0aa9";
```

Comment:
You can do "v${version}" as the revision here (fetchFromGitHub takes the information provided and turns it into a URL to a tarball generated by github).